vortex-array/src/stats/mod.rs

```diff
@@ -225,7 +226,7 @@ where
 }
 
 impl dyn Statistics + '_ {
-    pub fn get_as<U: for<'a> TryFrom<&'a Scalar, Error = VortexError>>(
+    pub fn get_as<U: for<'a> TryFrom<&'a ScalarValue, Error = VortexError>>(
```

Comment:
Can you add a doc string to this? Before get_as and get_as_cast had different behaviour, I think get_as will now always cast? But it's not clear with the docs.